openmc/stats/multivariate.py

```diff
@@ -375,6 +377,103 @@ def from_xml_element(cls, elem):
         return cls(x, y, z)
 
 
+class SphericalIndependent(Spatial):
+    """Spatial distribution with independent r, theta, and phi distributions.
+
+    This distribution allows one to specify a coordinates whose r-, theta-, and
+    phi- components are sampled independently from one another.
```

Comment:
You can use ```:math:`r` ```, ```:math:`\theta` ```, and ```:math:`\phi` ``` to render the correct mathematical symbols in the documentation.